### PR TITLE
Add option to stop control on system time jump

### PR DESCRIPTION
**Pull request by at-wat**
Use yp-spur option added by https://github.com/openspur/yp-spur/pull/191

### CODE REVIEW

**Comment by at-wat** [comment]
## [108] PASSED on noetic

<details><summary>All tests passed</summary>


```
build/test_results/ypspur_ros/roslint-ypspur_ros.xml: 1 tests
build/test_results/ypspur_ros/rostest-test_test_joint_trajectory.xml: 1 tests
build/test_results/ypspur_ros/rosunit-test_joint_trajectory.xml: 2 tests
Summary: 4 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Review by Taka-Kazu — APPROVED** [review]

**Comment by at-wat** [comment]
I'll update yp-spur version constraint once https://github.com/openspur/yp-spur/pull/191 is merged and released

File: src/ypspur_ros.cpp [hunk 1]
```diff
@@ -98,6 +98,7 @@ class YpspurRosNode
   int key_;
   bool simulate_;
   bool wait_convergence_of_joint_trajectory_angle_vel_;
+  bool exit_on_time_jump_;
```
Comment (at-wat) [inline src/ypspur_ros.cpp:101]: This can be function local variable

**Comment by at-wat** [comment]
## [110] PASSED on noetic

<details><summary>All tests passed</summary>


```
build/test_results/ypspur_ros/roslint-ypspur_ros.xml: 1 tests
build/test_results/ypspur_ros/rostest-test_test_joint_trajectory.xml: 1 tests
build/test_results/ypspur_ros/rosunit-test_joint_trajectory.xml: 2 tests
Summary: 4 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Pull request merged**